### PR TITLE
Add script to check for missing tokens

### PR DESCRIPTION
**Pull request by lukasoppermann**
## Summary

As discussed with @langermank this script checks if there are any tokens missing in the themes compared to the default `light` theme.

### CODE REVIEW

File: scripts/diffThemes.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import {PrimerStyleDictionary} from '~/src/PrimerStyleDictionary'
+// @ts-expect-error: no types exists for style-dictionary createDictionary
+import createDictionary from 'style-dictionary/lib/utils/createDictionary'
+import {themes as themesConfigArray} from './themes.config'
+import type StyleDictionary from 'style-dictionary'
+
+PrimerStyleDictionary.registerTransform({
```
Comment (rezrah) [inline scripts/diffThemes.ts:7]: Should this be here? I thought we put our transforms in a dedicated folder or is this a transient config change only needed for the diff?

**Review by rezrah — APPROVED** [review]
✅ to help move this forward.

Left a few suggestions / questions, but nothing blocking.

File: scripts/diffThemes.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }),
+  ]
+})
+
+/**
+ *
+ * @param mainThemeName - name of the theme (from themes.config.ts) to compare all other themes against
+ */
+
+const diffThemes = (mainThemeName: string) => {
```
Comment (rezrah) [inline scripts/diffThemes.ts:40]: Would be cool to have some tests for this with happy/unhappy path data to confirm it works as expected.. but from scanning the code it looks good 👍

File: scripts/diffThemes.ts (hunk 1 above)
Comment (lukasoppermann) [reply scripts/diffThemes.ts:7]: Actually, good spotting, this is a leftover, I ended up not needing it.

It used to be for this diff only though.

File: scripts/diffThemes.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  const mainIndex = themes.findIndex(([name]) => name === mainThemeName)
+  const mainTokens: string[] = themes.splice(mainIndex, 1)[0][1] as string[]
```
[the remark this answers is not in the document]
Comment (lukasoppermann) [reply scripts/diffThemes.ts:42]: Yes, good point, with using a local copy. I don't think the index is brittle though, given that we get the index by theme name before. But I will rework this.

File: scripts/diffThemes.ts (hunk 2 above)
Comment (lukasoppermann) [reply scripts/diffThemes.ts:40]: Yes, I will look into this as a separate PR. 👍

**Pull request merged**